### ISSUE TITLE
Add Security Groups IDs as output

### ISSUE THREAD

**Issue by seriousben** [docker/for-aws#54]
### Expected behavior

It would be useful to have the Manager and SwarmWide Security Group IDs as output especially when using terraform where the security group IDs could be used as input to other AWS resources.

### Information

I was able to make this work with some jq magic:

```bash
#!/bin/bash
url=https://editions-us-east-1.s3.amazonaws.com/aws/edge/Docker-no-vpc.tmpl

jq_script='
  .Outputs.ManagerSecurityGroupID = {"Description": "SecurityGroup ID of ManagerVpcSG", "Value": {"Ref": "ManagerVpcSG"}} |
  .Outputs.SwarmWideSecurityGroupID = {"Description": "SecurityGroup ID of SwarmWideSG", "Value": {"Ref": "SwarmWideSG"}}
'

curl -sSL $url | jq "$jq_script" | jq "." > Docker.tmpl
```

**Comment by kencochrane** [comment]
This has been added, and will be available in 17.07

**Comment by seriousben** [comment]
Thanks :)
